src/mnxmeth.c

```diff
@@ -169,7 +169,7 @@ int gettitleheight(LispPTR medleywin)
 
 /**************************************************************/
 /* setlineattributes                                          */
-/* This function attempts to syncronize the line data in lisp */
+/* This function attempts to synchronize the line data in lisp */
```

Comment:
In my local typo branch, I had line-wrapped `lisp` down to the next line and turned it to `Lisp`.

Reply:
I've made that change.